README.md

```diff
@@ -49,45 +49,44 @@ This is used by `rpm-ostree ex apply-live`.
 ## module "container": Encapsulate ostree commits in OCI/Docker images
 
 
-### Bundle an OSTree repository into a container
+### Bundle an OSTree repository into an OCI container directory
 
 Given an OSTree repository, running *outside* a container:
 
 ```
-$ ostree-container build --repo=/path/to/repo --ref=exampleos/x86_64/stable --oci-dir=/output/exampleos
+$ ostree-ext-cli container export-oci --repo=/path/to/repo exampleos/x86_64/stable /output/exampleos
 ```
-
-`--oci-dir` creates an [OpenContainers image](https://github.com/opencontainers/image-spec/blob/master/spec.md) layout.
-
 You can then e.g.
 
 ```
 $ skopeo copy oci:/output/exampleos containers-storage:localhost/exampleos
 $ podman run --rm -ti --entrypoint bash localhost/exampleos
 ```
 
-Another option is `--push quay.io/exampleos/exampleos:stable` which would push directly to a registry.  This would particularly be intended to be usable inside a fully unprivileged container, just mounting in the secrets necessary to push to the target registry.
+You can also use e.g. `skopeo copy oci:/output/exampleos docker://quay.io/exampleos/exampleos:latest`.
 
-### Take an arbitrary container and convert it to be OSTree ready
+### Maybe: Take an arbitrary container and convert it to be OSTree ready
 
 There's nothing conceptually stopping us from having tooling that takes
 an arbitrary container image and just makes it "ostree ready".  Or even
 just dyanamically accepting a container image that has a kernel client side.
 
 This *may* be in scope at some point in the future.
 
-#### ostree-containers and derivation
-
-For an ostree-based OS that is derived from Fedora, 
-`ostree-container build --from=registry.fedoraproject.org/fedora:33` would cause the generated container image to derive from the parent; in particular we de-duplicate content in the ostree commit from the base.
+#### Maybe: ostree-containers and derivation
```

Comment:
I wasn't sure if this is meant to be a subsection of "Maybe: Take an arbitrary container and convert it to be OSTree ready". (There's an additional `#`)

Reply:
OK I think it's clearer if that stuff lives in issues, not as "Maybe:" in the README.md.